backends/tofino/LICENSE

```diff
@@ -0,0 +1,10 @@
+Copyright 2013-2024 Intel Corporation.
```

Comment:
@hanw @grg Is this a new license or is it just outdated?

Reply:
It's outdated, will fix.